cli/pkg/credentialmanager/handler.go

```diff
@@ -171,12 +171,14 @@ func getCurrentContextFromKubeConfig() error {
 
 	fileContent, err := file.ReadFile(kubeconfig)
 	if err != nil {
-		return err
+		logging.PrintLog("Warning: could not open KUBECONFIG file: "+err.Error(), logging.InfoLevel)
```

Comment:
I'm not sure if I was doing it wrong, but I can't get this log message to be displayed. Should this be shown if I do the following
```
export KUBECONFIG=/a/b/c/d.yaml
./keptn version -v
```
?

Reply:
Just checked it and you are right - that message is not displayed. I have just committed a change to display this message via `stderr`